components/core/titles/LandingH2.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="flex justify-center">
+        <h2 :class="classObject">{{ subTitle }}</h2>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'LandingH2',
```

Comment:
This doesn't require any changes, but I personally would rename this component with a general term such as "CoreH2".

Reply:
I agree with you!